ValheimPlus/GameClasses/EnvMan.cs

```diff
@@ -25,41 +25,91 @@ public static void SetupDayLength()
             }
         }
 
+        private const float startOfADay = 0.25f;
```

Comment:
Where did we get these values from?

Reply:
Its "magic number" wrote on the original methods, and they don't have constant :-/

Reply:
All good just curious!